test/cpp/jit/test_gpu.cpp

```diff
@@ -10964,24 +10964,6 @@ TEST(NVFuserTest, FusionIssue363_CUDA) {
       &fusion, cg_outputs, aten_inputs, {aten_output}, __LINE__, __FILE__);
 }
 
```

Comment:
@naoyam Would appreciate your comment on removing test below. I believe most of the code that concerns #477 has drifted away. But we could definitely put something back here if we still need to watch out for similar scenario. 

Some context on why needed to work with this test:

This PR tries to move back the computeAt position if it is on the right of a broadcast iterdomain. So the `computeAtPosition` that this test checks will be moved left to 0.

Reply:
I think it's fine to remove it, but please add C++ tests to make sure computeAt positions are adjusted as such. No need to use the same math as this test.

Reply:
I see. Added a test that directly tests the compute and produce positions. Thanks!